Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -98,7 +98,7 @@ final class AttributeTests: XCTestCase {
       @1️⃣rethrows
       protocol P { }
       """,
-      substructure: Syntax(TokenSyntax.identifier("rethrows")),
+      substructure: Syntax(TokenSyntax.keyword(.rethrows)),
```

Comment:
Just curious, what changed the behavior?

Reply:
Oh that was because I forgot to check `remapping` in `consume(if:)`. Fixed now.